docs/containers-registries.conf.5.md

```diff
@@ -10,6 +10,8 @@ The CONTAINERS-REGISTRIES configuration file is a system-wide configuration
 file for container image registries. The file format is TOML.
 
 By default, the configuration file is located at `/etc/containers/registries.conf`.
+If the default configuration file is not set in system context, will use user created file at `$HOME/.config/containers/registries.conf`. If user's configuration file does not exist, will use `/etc/containers/registries.conf`
+as system-wide configuration path.
```

Comment:
suggest:
"If running as a rootless user, the file at `$HOME/.config/containers/registries.conf` will be used.  If that file does not exist, then the default configuration file will be used instead."

Reply:
This code does not contain any “running as rootless user” check (and that’s good!).